pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -483,24 +480,6 @@ func getGroupWrapper(group *kb.Group) *GroupWrapper {
 	}
 }
 
-func getMappedState(state kb.State) State {
-	switch state {
-	case kb.PASS:
-		return Pass
-	case kb.FAIL:
-		return Fail
-	case kb.WARN:
-		return Fail
-	case kb.INFO:
-		return NotApplicable
-	case SKIP:
-		return Skip
-	case NA:
-		return NotApplicable
-	}
-	return Fail
-}
-
```

Comment:
Can we keep this method commented, this gives us an easy reference of how we translate kube-bench states to our report.

Reply:
Can we include this in soime form of documentation instead? Like a package doc?